src/make_depend.rs

```diff
@@ -47,17 +47,19 @@ pub fn make_dependency_makefile(opts: crate::opts::MakeDepend) -> Result<(), Box
     let npm_pack_archive_dependencies = &internal_dependencies_exclusive
         .iter()
         .map(|dependency| {
-            let target = package_manifest
+            let target_directory = package_manifest
                 .directory()
-                .join(".internal-npm-dependencies")
+                .join(".internal-npm-dependencies");
+            let target = target_directory
                 .join(dependency.npm_pack_file_basename())
                 .to_str()
                 .expect("npm pack filename is not UTF-8 encodable")
                 .to_owned();
-            let source = dependency
-                .npm_pack_filename()
+            let source_package_directory = dependency.directory();
+            let source = diff_paths(source_package_directory, target_directory)
+                .expect("no relative path to source package")
                 .to_str()
-                .expect("npm pack filename is not UTF-8 encodable")
+                .expect("source package path is not UTF-8 encodable")
```

Comment:
`expect` messages start with an uppercase letter in this project, except for npm because npm is a proper noun 😅 

Not blocking, can follow-up